feature-layer-update-attributes/src/main/java/com/esri/arcgisruntime/sample/featurelayerupdateattributes/MainActivity.java

```diff
@@ -67,84 +79,186 @@ protected void onCreate(Bundle savedInstanceState) {
+        }
+
+        // update the Attributes map with the new selected value for "typdamage"
+        mSelectedArcGISFeature.getAttributes().put("typdamage", typeDamage);
+
+        try {
+          // update feature in the feature table
+          ListenableFuture<Boolean> mapViewResult = mServiceFeatureTable.updateFeatureAsync(mSelectedArcGISFeature);
+
+          // if successful, update change to the server
+          if (mapViewResult.get().booleanValue()) {
+
+            // apply change to the server
+            final ListenableFuture<List<FeatureEditResult>> serverResult = mServiceFeatureTable.applyEditsAsync();
+
+            serverResult.addDoneListener(new Runnable() {
+              @Override public void run() {
+                try {
+                  // check if server result successful
+                  List<FeatureEditResult> edits = serverResult.get();
+                  if (edits.size() > 0) {
+                    if (!edits.get(0).hasCompletedWithErrors()) {
+                      Log.e(getResources().getString(R.string.app_name), "Feature successfully updated");
+                    }
+                  } else {
+                    Log.e(getResources().getString(R.string.app_name), "The attribute type was not changed");
+                  }
+                  // display the callout with the updated value
+                  showCallout((String) mSelectedArcGISFeature.getAttributes().get("typdamage"));
+                } catch (Exception e) {
+                  Log.e(getResources().getString(R.string.app_name), "applying changes to the server failed: " + e.getMessage());
+                }
+
+              }
+            });
           }
-        });
-        return super.onSingleTapConfirmed(e);
+
+        } catch (Exception e) {
+          Log.e(getResources().getString(R.string.app_name), "updating feature in the feature table failed: " + e.getMessage());
+        }
       }
     });
 
+  }
+
+  /**
+   * Displays Callout
+   * @param title
+   */
+  private void showCallout(String title){
+
+    // create a text view for the callout
+    RelativeLayout calloutLayout = new RelativeLayout(getApplicationContext());
+
+    TextView calloutContent = new TextView(getApplicationContext());
+    calloutContent.setId(R.id.textview);
+    calloutContent.setTextColor(Color.BLACK);
+    calloutContent.setTextSize(18);
+    calloutContent.setPadding(0,10,10,0);
+
+    calloutContent.setText(title);
+
+    RelativeLayout.LayoutParams relativeParams = new RelativeLayout.LayoutParams(
+        RelativeLayout.LayoutParams.WRAP_CONTENT, RelativeLayout.LayoutParams.WRAP_CONTENT);
+    relativeParams.addRule(RelativeLayout.RIGHT_OF, calloutContent.getId());
+
+    // create image view for the callout
+    ImageView imageView = new ImageView(getApplicationContext());
+    imageView.setImageDrawable(getResources().getDrawable(R.drawable.ic_info_outline_black_18dp));
```

Comment:
`getDrawable()` is deprecated and will result in a logcat warning.  Refactor to use the AppCompat library: 

```java
ContextCompat.getDrawable(getActivity(), R.drawable.ic_info_outline_black_18dp);
```